packages/nimble-components/src/table/components/group-row/styles.ts

```diff
@@ -17,13 +17,20 @@ import { themeBehavior } from '../../../utilities/style/theme';
 import { userSelectNone } from '../../../utilities/style/user-select';
 
 export const styles = css`
-    ${display('flex')}
+    ${display('grid')}
 
     :host {
         align-items: center;
         height: calc(${controlHeight} + 2 * ${borderWidth});
         border-top: calc(2 * ${borderWidth}) solid ${applicationBackgroundColor};
         box-sizing: border-box;
+        grid-template-columns:
+            var(--ni-private-table-group-row-multi-select-size)
+            calc(
+                ${controlHeight} *
+                    (var(--ni-private-table-group-row-indent-level) + 1)
+            )
+            1fr;
     }
 
```

Comment:
The `controlHeight.getValueFor(this)` is still not a computation on control height, it's a conditional usage. We can express that in CSS:

```css
:host {
    grid-template-columns:
            calc(${controlHeight} * (var(--ni-private-table-group-row-indent-level) + 1) )
            1fr;
}

.host(.selectable) {
    grid-template-columns:
            var(--ni-private-table-group-row-multi-select-size)
            calc(${controlHeight} * (var(--ni-private-table-group-row-indent-level) + 1) )
            1fr;
}
```

Reply:
Oh duh...sometimes I just have blinders on.